Add a place confirmation email for SR2025 teams (#190)
srobo/team-emails: SR2025/2024-09-11-confirm-places.md

This year we hope to send out kits shortly before Kickstart, in early October.

In order to enable us to send out your kit you need to agree to our _kit
disclaimer_ (PDF attached). This disclaimer covers the terms and conditions
under which you may use the kit and outlines the limits of our responsibilities.
Please **sign it, scan it and email it back to us**.

We will be sending your kit to the shipping address you provided when you signed
up:

    SHIPPING_ADDRESS

We will send you an email with a tracking link once your kit is on its way.

## Discord

To support your team throughout the year and enable you to share experiences
with other teams we have a [Discord][discord] server. This is the primary
communication method for competitor support, enabling competitors to get support
directly from our volunteer mentors at any time.

Your invite link is <https://discord.com/invite/TODO_REPLACE_DISCORD_LINK_BEFORE_SENDING>.
If you don't already have a Discord account, you can register an account for
free. Accounts should be per individual and not shared with others.

Within Discord there are a number of shared channels, which everyone has access

> to, as well as a channel which is private to your team. Your team's channel is

Reviewer: ```suggestion
to, as well as a channel which is just for your team, which can only be accessed by your team members and our Blueshirt volunteers. Your team's channel is
```

Reviewer 2: ```suggestion
to. Additionally there is a channel which is just for your team, which can only be accessed by your team members and our Blueshirt volunteers. Your team's channel is
```

> Within Discord there are a number of shared channels, which everyone has access
> to, as well as a channel which is private to your team. Your team's channel is
> called `team-TODO_REPLACE_TLA_BEFORE_SENDING`.

Author: ```suggestion
Within Discord there are a number of shared channels, to which everyone has
access. You also have a channel which is accessible only by you, your team and
our Blueshirt volunteers. Your team's channel is called
`team-TODO_REPLACE_TLA_BEFORE_SENDING`.
```
